src/views/virtualmachines/tree/VirtualMachineTreeView.scss

```diff
@@ -1,6 +1,10 @@
 .vms-tree-view {
   overflow-x: hidden;
 
+  .pf-v5-c-tree-view__list-item {
+    position: relative;
```

Comment:
this is the default do we need it? are they setting it differently?

Reply:
No, the tree items are `position: static`.